forgerock-openbanking-model/src/main/java/com/forgerock/openbanking/constants/OIDCConstants.java

```diff
@@ -68,6 +75,10 @@ public static TokenEndpointAuthMethods fromType(String type) {
                     return tokenEndpointAuthMethods;
                 }
             }
+            log.error("Type '{}' doesn't match any of the token endpoint auth methods {}, Caller {}",
+                    type,
+                    TokenEndpointAuthMethods.getAllTypes(),
+                    StackWalker.getInstance(StackWalker.Option.RETAIN_CLASS_REFERENCE).getCallerClass().getCanonicalName());
             throw new UnsupportedOIDCAuthMethodsException("Type '" + type + "' doesn't match any of the token endpoint auth methods " + TokenEndpointAuthMethods.getAllTypes());
```

Comment:
This class is called from different points of source and services. The log.error can help to identify more easy the origin error.